infrastructure/httpapi/handlers/bridges.go

```diff
@@ -135,63 +132,63 @@ func (handler *Bridges) ListActivities(ctx *fasthttp.RequestCtx) {
 	if queryArgs.Has("createdAt.lt") {
 		maybeCreatedAtLt, createdAtLtParseErr := parseTimeFilter(string(queryArgs.Peek("createdAt.lt")))
 		if createdAtLtParseErr != nil {
-			ctx.SetStatusCode(fasthttp.StatusBadRequest)
+			httpapi.BadRequest(ctx, errors.New("invalid createdAt.lt"))
+			return
+		}
```

Comment:
Shall we include the `createdAtLtParseErr` in the 400 response? Feel like that may be easier for users to debug.

And there are many other similar places in this PR.